include/aos/common/downloader/downloader.hpp

```diff
@@ -8,30 +8,45 @@
 #ifndef AOS_DOWNLOADER_HPP_
 #define AOS_DOWNLOADER_HPP_
 
+#include "aos/common/cloudprotocol/alerts.hpp"
 #include "aos/common/tools/enum.hpp"
 #include "aos/common/tools/error.hpp"
 #include "aos/common/tools/string.hpp"
 
 namespace aos::downloader {
 
 /**
- * Download content type.
+ * Download item.
  */
-class DownloadContentType {
-public:
-    enum class Enum { eService, eLayer };
-
-    static const Array<const char* const> GetStrings()
+struct DownloadItem {
```

Comment:
Not sure we need additional struct for 5 params. We don't have clear rule about max function params count. But for me more readable to call this particular method with 5 params then fill DownloadItem struct before.